### PR TITLE
[CI:DOCS] Pass secrets from the host down to internal podman containers

### PR DESCRIPTION
**Pull request by rhatdan**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note
Subscriptions from the host now pass through to internal containers on contrib buildah images.
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
The commit messages don't jive with the description.  Did you mean to glom all this together here?

**Comment by rhatdan** [comment]
Nope

**Review by flouthoc — APPROVED** [review]
LGTM

**Comment by flouthoc** [comment]
PR mentions this as `[CI:DOCS]` however this is changing the buildah image so `[CI:DOCS]` tag looks incorrect.

**Comment by rhatdan** [comment]
I don't believe there are any tests that test the buildahimages so I treated it as a Docs update, to save the planet.

**Comment by rhatdan** [comment]
@Luap99 PTAL

File: contrib/buildahimage/Containerfile
Comment (edsantiago) [inline contrib/buildahimage/Containerfile:69]: Not a blocker, but is there any reason for doing it this way instead of creating a `mounts.conf` file in this subdirectory and adding it to the `ADD` on line 66?

**Comment by TomSweeneyRedHat** [comment]
LGTM
I'll let @Luap99 push the merge button.

File: contrib/buildahimage/Containerfile
Comment (rhatdan) [reply contrib/buildahimage/Containerfile:69]: Nope, but not sure that would be much better.

**Pull request merged**
